.gitmodules

```diff
@@ -13,6 +13,9 @@
 [submodule "puppet/git_modules/htpasswd"]
 	path = puppet/git_modules/htpasswd
 	url = https://github.com/ghoneycutt/puppet-htpasswd
+[submodule "puppet/git_modules/motd"]
```

Comment:
This is now a git submodule, but in the forklift bats test we install from the forge. That means we should release it to the forge, which I'll prepare. Then we can add it to the Puppetfile instead since the goal is to get rid of the submodules.